submissions/VasylRosokha/Popup/css/style.css

```diff
@@ -136,7 +141,7 @@ li {
     .menu, .more-menu {
         width: 300px;
     }
-    .menu-list {
+    .menu-list, .more-list {
         gap: 5px;
     }
 }
```

Comment:
Fo the future put an empty line at the end of every file. [Reason](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline?noredirect=1&lq=1). Tune your code editor's settings so it does this for you.